launch

```diff
@@ -70,18 +70,12 @@ def import_additional():
 import_additional()
 
 def install_nodejs():
-  if os_name == 'Darwin':
```

Comment:
why remove MAC related lines?
on MAC there is no wget https://www.mkyong.com/mac/wget-on-mac-os-x/

Reply:
Restored